app/src/main/AndroidManifest.xml

```diff
@@ -55,6 +55,9 @@
     <uses-permission android:name="android.permission.USE_BIOMETRIC" />
 
     <!-- This is needed to query content providers in other users -->
+    <uses-permission
+        android:name="android.permission.INTERACT_ACROSS_USERS"
```

Comment:
`_FULL` which we already have below is defined as the fuller form of this, so it's fine.

Reply:
Lint complained when I didn't include this

Reply:
Yes I meant adding this permission is fine if needed, but I'd just ignore Lint if it actually works.

Reply:
Sure, but someone will have to merge while bypassing it since otherwise it shows build failed below

Reply:
No I mean you'd actually have it ignore it, by an override

Reply:
Added `@Supress`